evm_loader/program/src/instruction/transaction_step.rs

```diff
@@ -38,7 +38,13 @@ pub fn do_begin<'a>(
     account_storage.block_accounts(true);
 
     let mut backend = ExecutorState::new(account_storage);
-    let evm = Machine::new(trx, caller, &mut backend)?;
+    let evm = Machine::new(
+        trx,
+        caller,
+        &mut backend,
+        #[cfg(feature = "tracing")]
+        None,
+    )?;
```

Comment:
This file should not be compiled when neon-evm is built as a library.

Reply:
Everything in `evm_loader/program/src/instruction` should excluded from the library build

Reply:
I agree, but for this we need either a new module or a new feature inside the existing module. I prefer a new module though.

This would make this PR even bigger and it will take time. I would postpone it for another PR.